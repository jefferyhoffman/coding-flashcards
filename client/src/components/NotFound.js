import React from 'react';

export default ({ location }) => {
  return (
    <div className="NotFound">
      <h3>Nothing found for path <code>{location.pathname}</code></h3>
    </div>
  );
}
