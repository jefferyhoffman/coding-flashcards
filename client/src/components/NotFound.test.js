import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';

describe('<NotFound>', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<NotFound location="/invalid/path" />);
    expect(wrapper).toMatchSnapshot();
  });
});
