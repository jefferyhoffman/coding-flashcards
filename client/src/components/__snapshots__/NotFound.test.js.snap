// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound> renders correctly 1`] = `
<div
  className="NotFound"
>
  <h3>
    Nothing found for path 
    <code />
  </h3>
</div>
`;
